README.md

```diff
@@ -108,7 +108,50 @@ Migration guide:
 This function is deprecated. Use the `torch.nn.attention.sdpa_kernel` context manager instead.
 
 Migration guide:
-Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
+
+Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. 
+If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
```

Comment:
This sentence doesn't make sense to me. "added to the input list of `sdpa_kernel`" - but we are removing `sdp_kernel`.